.travis.yml

```diff
@@ -1,5 +1,7 @@
 language: node_js
 node_js:
+  - "4.1"
+  - "4.0"
```

Comment:
feel free to remove iojs from the list.  You can do `"4"` instead of both 4.0 and 4.1